test/conformance/virtual_memory/urPhysicalMemRelease.cpp

```diff
@@ -8,8 +8,23 @@ using urPhysicalMemReleaseTest = uur::urPhysicalMemTest;
 UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urPhysicalMemReleaseTest);
 
 TEST_P(urPhysicalMemReleaseTest, Success) {
+    uint32_t referenceCount = 0;
+    ASSERT_SUCCESS(
+        urPhysicalMemGetInfo(physical_mem, UR_PHYSICAL_MEM_INFO_REFERENCE_COUNT,
+                             sizeof(referenceCount), &referenceCount, nullptr));
+    ASSERT_GE(1, referenceCount);
```

Comment:
this allows `referenceCount` to be 0, do we want that? also applies to the equivalent `Retain` test below

Reply:
this was something mentioned in an earlier comment https://github.com/oneapi-src/unified-runtime/pull/2272#discussion_r1830927816

Reply:
my mistake, should be `ASSERT_GE(referenceCount, 1)`

Reply:
corrected this